coordinator/src/services/request-builder.ts

```diff
@@ -0,0 +1,39 @@
+import Mustache from "mustache"
+import fs from "fs"
+import moment from "moment"
+import path from "path"
+import * as uuid from "uuid"
+
+const ebxmlRequestTemplate = fs.readFileSync(path.join(__dirname, "../resources/ebxml_request.mustache"), "utf-8")
+const cpaIdMap = new Map<string, string>(JSON.parse(process.env.CPA_ID_MAP))
+
+class EbXmlRequest {
+    timestamp = moment.utc().format()
+    conversation_id = uuid.v4().toUpperCase()
+    message_id = uuid.v4().toUpperCase()
+    from_party_id = process.env.FROM_PARTY_KEY
+    to_party_id = process.env.TO_PARTY_KEY
+    service = "urn:nhs:names:services:mm"
+
+    //These are the parameters for async reliable, as this is the only messaging pattern we need for EPS.
```

Comment:
Is it worth switching this to a todo so we can find it again afterwards?

Reply:
I don't think we'll need to find it again. Maybe I'll just get rid of the comment?

Reply:
Discussed - we may need to change this when we switch to using polling endpoints.